src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssStyleObject = {};
+  const cssStyles = sourceString.split(';');
+
+  cssStyles.forEach((style) => {
+    const [key, value] = style.trim().split(':');
```

Comment:
The `split(':')` method will not handle cases where the CSS property or value contains colons. Consider using a more robust parsing method if such cases need to be supported.